apps/daimo-mobile/src/view/screen/onboarding/OnboardingScreen.tsx

```diff
@@ -116,8 +112,14 @@ export default function OnboardingScreen({
     setName("");
     setInviteLink(undefined);
     createReset && createReset();
+    setStartedCreating(false);
   };
 
+  useEffect(() => {
+    if (page === "create") setStartedCreating(true);
+    if (page === "intro") setStartedCreating(false);
+  }, [page]);
```

Comment:
hmmm adding a latching state machine thing on which onboarding page you're on feels like... more moving parts.

full list
"create"
  | "existing"
  | "new-allow-notifications"
  | "existing-allow-notifications"
  | "new-loading";
  
  could do something stateless like this?
  
  ```
  const startedCreating = page !== "create" && !page.startsWith("new-")
  ```

Reply:
fair, done